obsei/analyzer/classification_analyzer.py

```diff
@@ -34,25 +34,42 @@ def __init__(self, **data: Any):
             device=self._device_id,
         )
 
-        if hasattr(self._pipeline.model.config, 'max_position_embeddings'):
+        if hasattr(self._pipeline.model.config, "max_position_embeddings"):
             self._max_length = self._pipeline.model.config.max_position_embeddings
         else:
             self._max_length = 510
 
     def _classify_text_from_model(
-        self, text: str, labels: List[str], multi_class_classification: bool = True
-    ) -> Dict[str, float]:
-        scores_data = self._pipeline(
-            text[: self._max_length] if len(text) > self._max_length else text,
+        self,
+        texts: List[str],
+        labels: List[str],
+        multi_class_classification: bool = True,
+    ) -> List[Dict[str, float]]:
+        if len(texts) <= 1:
```

Comment:
Why this condition? it seems it is not required

Reply:
How about this then -

```
prediction = self._pipeline(
                    texts,
                    labels,
                    multi_label=multi_class_classification
)
return prediction if isinstance(prediction, list) else [prediction]
```

Reply:
@shahrukhx01 I think this one last pending :)

Reply:
@lalitpagaria I missed this one. Done now!